Description: fixes for -Werror=format-security.
 Source fails to build with -Werror=format-security because a function return
 value is being passed to the visual_log() function where a format string is
 expected.  This isn't a security hole because the strings returned aren't
 based on untrusted input, but we should clean up anyway.
Author: Steve Langasek <vorlon@debian.org>
Last-Update: 2011-10-26
Index: libvisual-0.4.0/libvisual/lv_plugin.c
===================================================================
--- libvisual-0.4.0.orig/libvisual/lv_plugin.c
+++ libvisual-0.4.0/libvisual/lv_plugin.c
@@ -442,11 +442,11 @@ VisList *visual_plugin_registry_filter (
 				
 				visual_list_add (list, ref);
 			} else if (ret != FALSE) {
-				visual_log (VISUAL_LOG_WARNING, visual_error_to_string (ret));
+				visual_log (VISUAL_LOG_WARNING, "%s", visual_error_to_string (ret));
 			}
 		}
 		else if (ret != FALSE) { /* FIXME XXX TODO, patch frmo duilio check how this works */
-			visual_log (VISUAL_LOG_WARNING, visual_error_to_string (ret));
+			visual_log (VISUAL_LOG_WARNING, "%s", visual_error_to_string (ret));
 		}
 	}
 
